### PR TITLE
Update statistics integration with mean_circular

### PR DESCRIPTION
**Pull request by enzo2**
## Proposed change

Documentation for proposed feature - https://github.com/home-assistant/core/pull/98930

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/98930

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/statistics.markdown [hunk 1]
```diff
@@ -52,6 +52,7 @@ The following are supported for `sensor` source sensors `state_characteristic`:
 | `distance_99_percent_of_values` | A statistical indicator derived from the standard deviation of an assumed normal distribution. 99% of all stored values fall into a range of returned size.
 | `distance_absolute` | The difference or "spread" between the extreme values of measurements. Equals `value_max` minus `value_min`.
 | `mean` | The average value computed for all measurements. Be aware that this does not take into account uneven time intervals between measurements.
+| `mean_circular` | The [circular mean](https://en.wikipedia.org/wiki/Circular_mean) for angular measurements (_e.g._ direction). Assumes that measurements are expressed in degrees.
```
Comment (ThomDietrich) [inline source/_integrations/statistics.markdown]:
```suggestion
| `mean_circular` | The [circular mean](https://en.wikipedia.org/wiki/Circular_mean) for angular measurements (_e.g._ wind direction). Assumes that measurements are expressed in degrees, e.g., 180° or -90°.
```

**Review by ThomDietrich — APPROVED** [review]

**Review by ThomDietrich — APPROVED** [review]

**Comment by enzo2** [comment]
Thinking about negative inputs, I realized we may want to specify that the output will be positive. Since arctan2 returns between [-π, π] (-180 to 180°), I shifted the output to 0 - 360° according to the usual convention for polar coordinates. So we may want to note in the documentation that while it will understand negative measurements, it will output positive ones.

**Comment by ThomDietrich** [comment]
Good call. Please do, as short as needed :)

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**